integration-tests/join/jail.nix

```diff
@@ -0,0 +1,104 @@
+with import <nixpkgs> {};
```

Comment:
isn't this file identical to https://github.com/crypto-com/chain/blob/master/integration-tests/jail/jail.nix ? why not to reuse that / import that?

Reply:
i'll change it